sponge/src/main/java/com/griefcraft/lwc/LWCPlugin.java

```diff
@@ -0,0 +1,26 @@
+package com.griefcraft.lwc;
+
+import com.google.inject.Inject;
+import org.slf4j.Logger;
+import org.spongepowered.api.event.Listener;
+import org.spongepowered.api.event.game.state.GameStartedServerEvent;
+import org.spongepowered.api.event.game.state.GameStoppedServerEvent;
+import org.spongepowered.api.plugin.Plugin;
+
+@Plugin(id = "lwc", name = "LWC", version = "1.0")
```

Comment:
So I'm sure there is a way that we could put ${project.version} in here and have it be replaced as part of compilation.

I'm unsure how as of yet, but hey 🤷‍♂

Reply:
Haven't figured it out yet. I'll try digging in deeper in the morning.